core/server/src/main/java/alluxio/master/file/FileSystemMaster.java

```diff
@@ -479,19 +479,20 @@ public Status getStatus() {
   /**
    * @return the status of the startup consistency check and inconsistent paths if it is complete
    */
-  public StartupConsistencyCheck getStartupConsistencyCheck() {
+  public StartupConsistencyCheckResult getStartupConsistencyCheck() {
     if (!Configuration.getBoolean(PropertyKey.MASTER_STARTUP_CONSISTENCY_CHECK_ENABLED)) {
-      return new StartupConsistencyCheck(StartupConsistencyCheck.Status.DISABLED, null);
+      return new StartupConsistencyCheckResult(StartupConsistencyCheckResult.Status.DISABLED, null);
     }
     if (!mStartupConsistencyCheck.isDone()) {
-      return new StartupConsistencyCheck(StartupConsistencyCheck.Status.RUNNING, null);
+      return new StartupConsistencyCheckResult(StartupConsistencyCheckResult.Status.RUNNING, null);
```

Comment:
Are you suggesting to have something like `StartupConsistencyCheckResult.FAILURE` as a constant and only have a constructor for the `COMPLETE` case? Another approach I was thinking about was just moving all the logic into the class, but I think its nicer to use it as a data structure.